### PR TITLE
CI: add CI to test nim compiler compiles and is the correct location

### PR DESCRIPTION
**Pull request by ynfle**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # 2.4.0
+
+    - name: Build Image
+      run: docker build -t "exercism/nim-docker-base:${NIM_VERSION}" .
+
+    - name: Create container with test entrypoint 
+      run: docker create --rm --entrypoint /nim/bin/nim --name ndb "exercism/nim-docker-base:${NIM_VERSION}" r --eval:'import os; echo getFileInfo("/nim/bin/nim")'
+
+    - name: Run container
+      run: docker start -a ndb
```
Comment (ErikSchierboom) [inline .github/workflows/ci.yml:29]: You're probably aware, but just in case: I think you can combine these two commands into a single `docker run`

**Review by ErikSchierboom — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ynfle) [reply .github/workflows/ci.yml]: Fixed

**Comment by ynfle** [comment]
Caching doesn't seem to be working, unless I'm not understanding how it's supposed to work.

**Comment by ErikSchierboom** [comment]
> Caching doesn't seem to be working, unless I'm not understanding how it's supposed to work.

That's because your `docker build` command doesn't actually use the cache. The easiest way to solve this would be to use the `docker/build-push-action` action, but not push anything. This looks something like this: https://github.com/exercism/github-actions/blob/main/.github/workflows/docker-build-push-image.yml#L76-L83

You could also look at https://github.com/docker/build-push-action to see how they're using the cache.

**Comment by ynfle** [comment]
Sorry for the spam, I'm trying to figure out the caching

**Comment by ynfle** [comment]
I'm going to merge this, and working on caching in a future PR.

@ErikSchierboom Can this check be made mandatory to pass?

**Pull request merged**